steps/microsoft-teams-adaptive-card/1.0.4/step.yml

```diff
@@ -0,0 +1,162 @@
+title: Microsoft Teams Adaptive Card Integration
+summary: |
+  Send Microsoft Teams message to a channel
+description: |
+  Send Microsoft Teams message as adaptive card
+website: https://github.com/HUK-COBURG/bitrise-step-microsoft-teams-adaptive-card
+source_code_url: https://github.com/HUK-COBURG/bitrise-step-microsoft-teams-adaptive-card
+support_url: https://github.com/HUK-COBURG/bitrise-step-microsoft-teams-adaptive-card/issues
+published_at: 2025-01-31T12:19:02.662532+01:00
+source:
+  git: https://github.com/HUK-COBURG/bitrise-step-microsoft-teams-adaptive-card.git
+  commit: 85309d120949de3079a73509c9aa7b0afb3bfab3
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  go:
+    package_name: github.com/HUK-COBURG/bitrise-step-microsoft-teams-adaptive-card
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: true
+inputs:
+- is_debug_mode: "no"
+  opts:
+    description: |
```

Comment:
Please add `summary` key for all inputs. Can be deferred to a later Step release.